pdns/dnsparser.cc

```diff
@@ -582,7 +582,7 @@ class DNSPacketMangler
     : d_packet(packet), d_length(length), d_notyouroffset(12), d_offset(d_notyouroffset)
   {}
   
-  void skipLabel()
+  void skipDomainName()
```

Comment:
It would make sense to add a docblock above the function as well, so the next person who stumbles unto this does not have to figure out what this function _exactly_ does, regardless of the name

Reply:
[This](https://github.com/PowerDNS/pdns/blob/50294c448e545f7ba33756e2db1003be075b608f/pdns/syncres.cc#L154-L166) and [this](https://github.com/PowerDNS/pdns/blob/50294c448e545f7ba33756e2db1003be075b608f/pdns/syncres.cc#L1042-L1048) should give some guidance

Reply:
Thanks; done.